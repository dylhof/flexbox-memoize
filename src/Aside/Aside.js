import React from 'react';
import Info from '../Info/Info.js';

function Aside(props) {
  return (
    <aside className='aside'>
      <div className='aside-score-div'> 
        <h3>Your Score:</h3>
        <p>{props.correctAnswers} Correct Answers</p>
        <p>{props.answeredQuestions} Questions Answered</p>
      </div>
      <div className='aside-info-div'>
        <button className='learn-more-btn' >
          <a href='https://css-tricks.com/snippets/css/a-guide-to-flexbox/' target='_blank'>
            Learn more about flexBox!
          </a>
        </button>
        <Info 
          currentQuestion={props.currentQuestion}
          showInfo={props.showInfo}
          quizStarted={props.quizStarted}
          toggleShowInfo={props.toggleShowInfo}/>
      </div>
    </aside>
  )
}

export default Aside;