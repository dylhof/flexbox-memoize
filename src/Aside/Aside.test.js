import React from 'react';
import Aside from './Aside';
import { shallow } from 'enzyme';

describe('Aside', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
      <Aside />
    )
  });

  it('should match the snapshot', ())
})