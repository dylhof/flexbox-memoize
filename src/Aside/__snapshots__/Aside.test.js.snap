// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Aside should match the snapshot 1`] = `
<aside
  className="aside"
>
  <div
    className="aside-score-div"
  >
    <h3>
      Your Score:
    </h3>
    <p>
       Correct Answers
    </p>
    <p>
       Questions Answered
    </p>
  </div>
  <div
    className="aside-info-div"
  >
    <button
      className="learn-more-btn"
    >
      <a
        href="https://css-tricks.com/snippets/css/a-guide-to-flexbox/"
        rel="noopener noreferrer"
        target="_blank"
      >
        Learn more about flexBox!
      </a>
    </button>
    <Info />
  </div>
</aside>
`;
